sw-precache-config.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  runtimeCaching: [
+    {
+      urlPattern: /^https?:\/\/(.*?).cloudfront.net\/bundles\/(.*?).js$/,
+      handler: 'networkFirst'
```

Comment:
> Try to handle the request by fetching from the network. If it succeeds, store the response in the cache. Otherwise, try to fulfill the request from the cache. This is the strategy to use for basic read-through caching. It's also good for API requests where you always want the freshest data when it is available but would rather have stale data than no data.

There is also `cacheFirst`, which could also be worth to consider:
> If the request matches a cache entry, respond with that. Otherwise try to fetch the resource from the network. If the network request succeeds, update the cache. This option is good for resources that don't change, or have some other update mechanism.